XpressAI/xpipes
# Dataset from dir support

1. Added dataset reading from directory support to `xai_learning/training.py`. Uses skikitlearn LabelEncoder to convert folder names into labels, and cv2 to read the image.
2. Added opencv-python to  `xai_learning/requirements.txt`

**Test:**

1. Make an image dataset with the folder name as the class name and populate with images. Alternatively try this weather dataset from kaggle: https://www.kaggle.com/vijaygiitk/multiclass-weather-dataset
2. Open KerasTrainImageClassifier.xpipe
3. Supply the path to the ReadDataSet Component as a `literal string`, eg I downloaded the weather dataset and placed inside   `/Datasets/dataset/`
4. Run the xpipe. You should be able to train the model.
